profiles/common/modules/custom/tmgmt_poetry/inc/tmgmt_poetry.webservice.inc

```diff
@@ -338,6 +338,13 @@ function _tmgmt_poetry_replace_job_in_content($content, $job, $job_item) {
       if (!$dom->loadHTML($content)) {
         return FALSE;
       }
+
```

Comment:
The actual location of this code does not matter. Simplexml provides another way of accessing the underlying DOM and changes are applied on the same in-memory structure. I agree to avoid possible future confusion it's probably best to move the code to where you placed it.